src/main/java/com/nike/vault/client/VaultClient.java

```diff
@@ -313,6 +333,8 @@ protected Response execute(final HttpUrl url, final String method, final Object
      * @param response Response to parses the error details from
      */
     protected void parseAndThrowErrorResponse(final Response response) {
+        logger.debug(LOG_HTTP_RESPONSE_BODY_TMPL, gson.toJson(response.body()));
```

Comment:
Can we log the body without calling gson here?

If the body isn't JSON, this is going to throw an exception.  This would be self defeating because part of the reason for this logging is so we'd be able to figure out what was going on if it wasn't JSON.

Also, seems like the debug statement should move into the try block (not sure if response.body can throw IOException or not).